GVFS/GVFS.Common/ProductUpgrader.cs

```diff
@@ -70,6 +70,8 @@ protected ProductUpgrader()
         {
         }
 
+        public string InstallerLogsDirectory { get; set; } = ProductUpgraderInfo.LogDirectory;
```

Comment:
What is the reason for the `= ProductUpgraderInfo.LogDirectory;`?

`ProductUpgraderInfo.LogDirectory` is simply "Logs", and it appears that this property is supposed to be a path (you might even consider renaming to `InstallerLogsDirectoryPath` to make that clearer).

Reply:
This was meant to be the full path to the log directory.